Marlin/src/lcd/language/language_hu.h

```diff
@@ -22,18 +22,22 @@
 #pragma once
 
 /**
- * Hungarian
- *
- * LCD Menu Messages
- * See also http://marlinfw.org/docs/development/lcd_language.html
+ * Magyar
  *
+ * LCD Menü Üzenetek. Lásd még https://marlinfw.org/docs/development/lcd_language.html
+ * Marlin 2.0.x bugfix Magyar fordítása. A fordítást folyamatosan javítom és frissítem.
+ * A Magyar fordítást készítette: AntoszHUN
+ * Youtube csatornám: https://www.youtube.com/3dprintingantosz
+ * Ha szeretnél támogatni megteheted PayPal-on: https://www.paypal.me/3dprintingantosz
+ * De akár Patreon-on is: https://www.patreon.com/3dprintingantosz
  */
```

Comment:
Did you mean to add this to the main repo? I don't think we want to start adding paypal & patreon links for contributions or it could be a slippery slope of everyone wanting to add theirs.